antarest/study/service.py

```diff
@@ -96,6 +96,15 @@ def __init__(
         self.task_service = task_service
         self.areas = AreaManager(self.raw_study_service)
         self.config = config
+        self.callbacks: List[Callable[[str], None]] = []
```

Comment:
this is a callback called only on study deletion, so this should be made clear. "on_delete_callabacks" or something like that